mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-Data-driven-approach.asciidoc

```diff
@@ -47,10 +58,6 @@ image::images/image30_new.png[]
 
 image::images/image32_new.png[]
 
-*Prior to mrchecker-core-module version 5.6.2.1*
-
-Parameters that are passed into tests using the `@Parameters` annotation must be _Object[]_s
-
 === In the annotation:
```

Comment:
In this section screenshot with example needs to be updated, because it shows imports and annotations from JUnit4, but it should be JUnit5.
Please also check if annotation "Parameters" is proper for JUnit5